spec/index.bs

```diff
@@ -4377,9 +4365,9 @@ if their internal {{GPUPipelineLayout/[[bindGroupLayouts]]}} sequences contain
             }]
         });
 
-        const pipelineLayout = gpuDevice.createPipelineLayout({
+        const pipelineLayoutRequirements = {
             bindGroupLayouts: [bindGroupLayout]
-        });
+        };
```

Comment:
Maybe we should inline it, but that'd require an investigation of what are the other things that could go in `layout:` from Vulkan's Pipeline Layout, D3D12 Root Table Signature and MTLRenderPipeline (I guess? It doesn't have an equivalent of the layout).

Or `layout: { bindGroups: [....]}` but that's a bit werid because you don't put bindgroups in there.

Reply:
I think inlining it would be fine. If it turns out there's more stuff in the pipeline layout, we can always add sibling members. Nothing really says we have to encapsulate the pipeline layout requirements into one object.

Could be described by inheriting the GPUPipelineLayoutRequirements instead of having a member.

Reply:
Verbosity can be good if it leads a better understanding of the API, and how the different pieces interact. I think the trade-off is worth it in this case. And we might be adding more things to the pipeline layout in the future.